### DISCUSSION TITLE
Nominate @brayanhenao to the role of Maintainer

### DISCUSSION THREAD

**Discussion by thitch97** [paketo-buildpacks/feedback#235] [Node.js Team]
I'd like to nominate @brayanhenao as a maintainer for the Node.js subteam. He has a vested interest in the health of the Node.js buildpacks has already performed duties similar to that of a maintainer (e.g. [npm-install](https://github.com/paketo-buildpacks/npm-install/commits/main/?author=brayanhenao) & [node-engine](https://github.com/paketo-buildpacks/node-engine/commits/main/?author=brayanhenao)).

👍 / 👎 ?

cc/ @paketo-buildpacks/nodejs-maintainers
@paketo-buildpacks/steering-committee

**Comment by ForestEckhardt** [comment]
This has been approved! Congrats @brayanhenao.

Please open a PR on the [TEAMS.md](https://github.com/paketo-buildpacks/community/blob/main/TEAMS.md) to add @brayanhenao to the list of maintainers.
